packages/dds/legacy-dds/package.json

```diff
@@ -1,7 +1,6 @@
 {
-	"name": "@fluid-private/legacy-dds",
+	"name": "@fluidframework/legacy-dds",
 	"version": "2.42.0",
-	"private": true,
```

Comment:
With these changes, I believe we'll be publishing this package to NPM - just verifying that's intended